### ISSUE TITLE
inkonsistente Argumentreihenfolge

### ISSUE THREAD

**Issue by jvoigtlaender** [fmidue/flex-tasks#36]
Es ist merkwürdig (aber würde bei Anpassung nun natürlich Änderungen an vielen Stellen erfordern), dass gemäß https://github.com/fmidue/flex-tasks/blob/83ce9523b37323459c5f0982cf0fc16ca4fd5dd2/flex-tasks/src/FlexTask/DefaultConfig.hs#L137-L139 die Checker-Funktionen erst die Aufgabeninstanz-Daten, dann den Dateipfad nehmen sollen, während andernorts die `partialGrade` und `completeGrade` Funktionen es immer gerade umgekehrt tun und auch etwa in Autotool die Argumentfunktion für `toReporterWithGlobalPics` gerade andersherum gestrickt ist.

**Comment by patritzenfeld** [comment]
Ja, das ist mir gestern auch aufgefallen. Das stört ein wenig, weil man dann generell weniger Eta-Reduction bei "Glue-Tasks" verwenden kann.

Ich denke wir sollten das jetzt erstmal so lassen und ich werde es nach dem Semester anpassen, damit es keine Probleme mit den schon gestellten Aufgaben gibt.
